quote_generator.js

```diff
@@ -0,0 +1,17 @@
+const quote_generator = () => {
+    // Storing the API endpoint in a variable
+    const url = "https://type.fit/api/quotes/";
+    // Fetching the API
+    fetch(url)
+        .then((response) => response.json())
+        .then((data) => {
+            // Generate quotes
+            let RandomQuotes = Math.floor(Math.random() * data.length);
+            let quote = data[RandomQuotes].text;
+            console.log(quote);
+        })
+        .catch(() => {
```

Comment:
When the code goes to .catch statement it will give an undefined error because there is no err. The code should be something like this: 
```
.catch((err) => {
   if (err) throw err;
})
```